test/widgets/content_test.dart

```diff
@@ -158,6 +166,78 @@ void main() {
     });
   });
 
+  group('Internal links', () {
+    Future<List<Route<dynamic>>> prepareContentWithNavigator(WidgetTester tester, {
```

Comment:
This function name contrasts with others in the file by saying "with navigator". But it doesn't create any more of a navigator than those other functions called `prepareContent` — they each mount a `MaterialApp` widget, and that creates a `Navigator`, exactly like what happens in this function. So the name is misleading.

One thing this function does do is add a navigator *observer*. But that's a kind of observer, not a kind of navigator.

I think just `prepareContent` suffices as a name. The function is a local helper for this test group; so it prepares some content, and the specific set of bells and whistles involved in doing so is just whatever set this test group needs.